public/static/js/projects.js

```diff
@@ -902,8 +902,8 @@ mciModule.controller(
     };
 
       $scope.removeContainerSize = function (name) {
-          if ($scope.settingsFormData.container_sizes[name]) {
-              delete $scope.settingsFormData.container_sizes[name]
+          if ($scope.settingsFormData.container_size_definitions[name]) {
```

Comment:
I think I mentioned this in the Jira ticket, but since we don't plan on supporting container settings in the legacy project settings UI, we can remove the legacy container UI logic entirely.

Reply:
Sounds good, removing from JS/HTML and the `modifyProject` function